src/components/multi-language/languagesupport.js

```diff
@@ -0,0 +1,8 @@
+
+const languagesupport = ({ setCity }) => {
```

Comment:
For consistency in naming conventions, please change this to `LanguageSupport` and `MultiLanguage/LanguageSupport.js`.